### DISCUSSION TITLE
Nerdctl build --ssh on windows 11

### DISCUSSION THREAD

**Discussion by ahjulstad** [containerd/nerdctl#829] [Q&A]
Pointers on getting `nerdctl build --ssh default` working would be greatly appreciated, either in WSL or in plain CMD. I am on Windows 11 with Rancher Desktop v 1.0.1

Git with ssh-agent on WSL works, the same with git on windows, as long as I point git to the microsoft provided OpenSSH binary. However, nerdctl is never able to connect to ssh-agent socket. With default it complains about SSH_AUTH_SOCK not set (but it is), if I specify the socket manually, I get error message like `error: stat /tmp/ssh-mYRmExe8pB8b/agent.105: no such file or directory`. 

Not knowing exactly how these things works, does it have something to do with buildkit running someplace else, and cannot see the ssh-agent socket or the environment variable?

**Comment by AkihiroSuda** [comment]
Maybe better to submit a request in https://github.com/rancher-sandbox/rancher-desktop/issues ?

Rancher Desktop edition of `nerdctl.exe` is different from the upstream nerdctl: https://github.com/rancher-sandbox/rancher-desktop/tree/v1.0.1/src/go/nerdctl-stub

**Reply by ahjulstad** [reply]
Thanks for the pointer. [Submitted](https://github.com/rancher-sandbox/rancher-desktop/issues/1649).
